src/hope_dedup_engine/apps/faces/celery_tasks.py

```diff
@@ -68,51 +66,55 @@ def finish_with_success(ds: DeduplicationSet) -> None:
 @app.task(bind=True, base=DedupeTask, shadow_name=shadow_name)
 def encode_chunk(
     self: DedupeTask,
-    files: list[str],
-    config: dict[str, Any],
+    deduplication_set_id: UUID,
+    encoding_ids: list[UUID],
 ) -> None:
     """Encode faces in a chunk of files."""
-    with report_long_execution('DeduplicationSet.objects.get(pk=config.get("deduplication_set_id"))'):
-        ds = DeduplicationSet.objects.get(pk=config.get("deduplication_set_id"))
+    deduplication_set = DeduplicationSet.objects.get(pk=deduplication_set_id)
+    config = DeduplicationSetConfig.from_deduplication_set(deduplication_set)
     try:
-        failed_encodings: dict[str, Image.StatusCode] = {}
-        with report_long_execution("encode_faces(files, config, pre_encodings, progress=callback)"):
-            results = encode_faces(
-                files,
-                process_encoding_error=failed_encodings.__setitem__,
-                config=config,
-            )
+        encode_faces(
+            deduplication_set,
+            encoding_ids,
+            config.face_confidence_threshold,
+            config.deduplicate.model_name,
+            config.deduplicate.detector_backend,
+            align=config.deduplicate.align,
+        )
 
-        with report_long_execution("ds.update_encodings(results[0])"):
-            ds.update_encodings(results[0])
-        if failed_encodings:
-            ds.update_findings(
-                (filename, "", 0, status_code.value) for filename, status_code in failed_encodings.items()
-            )
     except Exception as e:
         sentry_sdk.capture_exception(e)
-        finish_with_error(ds, e)
+        finish_with_error(deduplication_set, e)
         raise
 
 
 @app.task(bind=True, base=DedupeTask)
 def dedupe_chunk(
     self: Task,
-    files0: list[str],
-    files1: list[str],
-    config: dict[str, Any],
-) -> FindingType:
+    deduplication_set_id: UUID,
+    encoding_ids0: list[UUID],
+    encoding_ids1: list[UUID],
+) -> None:
     """Deduplicate faces in a chunk of files."""
-    ds = DeduplicationSet.objects.get(pk=config.get("deduplication_set_id"))
+    ds = DeduplicationSet.objects.get(pk=deduplication_set_id)
```

Comment:
how do we can dedupe in chunks? Is the function name correct?
per Business logic we shoud deduplicate latest DeduplicationSet with all previous approved DeduplicationSets for the same DeduplicationGroup. 
This function is correct only if:
encoding_ids0 == lastest deduplcationSet or part of it
encoding_ids1 == all approved deduplicationsets